### PR TITLE
Move argparse phonesim code into module to be invoked as tool

### PR DESCRIPTION
**Pull request by Hexxeh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      author_email='katharine@pebble.com',
+      license='MIT',
+      packages=packages,
+      install_requires=requires,
+      entry_points={
+          'console_scripts': [
+            'pypkjs=runner.websocket:run_tool'
+          ],
+      },
+      zip_safe=True)
```
Comment (Katharine) [inline setup.py]: Is this true?

Comment (Hexxeh) [inline setup.py]: I'm not 100% sure, but I invoked pypkjs via PATH with a PBW and had the demo app pass.

Comment (Katharine) [inline setup.py]: I'm particularly concerned about the GeoIP database — what does getting location do?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  'sh==1.09',
+  'six==1.9.0',
+  'websocket-client==0.31.0',
+  'wsgiref==0.1.2',
+  'libpebble2==0.0.12',
+  'netaddr==0.7.18'
+]
+
+packages = find_packages()
+print packages
```
Comment (Katharine) [inline setup.py]: Debug line; you can probably remove it (and then perhaps inline the `find_packages()` call.

File: setup.py (hunk 1 above)
Comment (Hexxeh) [inline setup.py]: Not sure how to test that, but a search revealed the geoIP stuff isn't zip_safe, so I've changed this.

**Comment by Katharine** [comment]
You should probably include this bit:

```python
from gevent import monkey
monkey.patch_all()
```

In `runner/__init__.py`, too — the monkeypatch should be the _first_ thing done, and runner is importer before runner.websocket.

**Comment by Hexxeh** [comment]
Addressed the comments, happy to merge?

**Comment by Katharine** [comment]
I think it might be better if the value of `requires` in setup.py was generated from `requirements.txt`, to avoid duplicating the list (because I bet I won't remember to update both).

**Comment by Katharine** [comment]
:+1:

**Pull request merged**